cluster-assign-roles.sh

```diff
@@ -157,7 +115,7 @@ function hadoop_install {
   install_stub $(printf ${hosts// /\\n} | sort)
 
   printf "Assigning roles for headnodes...\n"
-  for h in $(printf ${hosts// /\\n} | grep -i "BCPC-Hadoop-Head" | sort); do
+  for h in $(printf ${hosts// /\\n} | grep -i "BCPC-Hadoop-Namenode" | sort); do
```

Comment:
Erm, this only runs for Namenodes but all comments and `printf`'s still say headnodes. Further, this will no longer run for headnodes which are not namenodes (e.g. a mapreduce head on a physical cluster). This works for the specific VM setup but not a general cluster from what I can tell.